cmd/s3-peer-client.go

```diff
@@ -187,9 +142,7 @@ func (s3p *s3Peers) SendRPC(peers []string, method string, args interface {
 // request to all peers. Currently we log an error and continue.
 func S3PeersUpdateBucketNotification(bucket string, ncfg *notificationConfig) {
 	setBNPArgs := &SetBNPArgs{Bucket: bucket, NCfg: ncfg}
-	peers := globalS3Peers.GetPeers()
-	errsMap := globalS3Peers.SendRPC(peers, "S3.SetBucketNotificationPeer",
-		setBNPArgs)
+	errsMap := globalS3Peers.SendUpdate(globalS3Peers.Peers, setBNPArgs)
```

Comment:
globalS3Peers.Peers need not be sent as SendUpdate itself can access it using s3p.Peers